ci/scripts/generate-dns-values.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+test $DNS_VAR_FILE
+
+PROMETHEUS_EXTERNAL_HOSTNAME=$(dig +short public.prometheus.monitoring.cld.internal ptr)
```

Comment:
Hmmm `dig` does not appear to return a non-zero code if it encounters an error, so using it might make things hard to debug... Is it possible to use sdget instead? It should be in the docker image.

Reply:
good point on non-zero exit code.  let me explore using sdget.